src/rsz/src/SizeDownMove.cc

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025-2025, The OpenROAD Authors
+
+#include "SizeDownMove.hh"
+
+#include <algorithm>
+#include <cmath>
+#include <string>
+
+#include "BaseMove.hh"
+
+namespace rsz {
+
+using std::pair;
+using std::string;
+using std::vector;
+
+using utl::RSZ;
+
+using sta::ArcDelay;
+using sta::Edge;
+using sta::Instance;
+using sta::InstancePinIterator;
+using sta::LibertyCell;
+using sta::LibertyPort;
+using sta::LoadPinIndexMap;
+using sta::Net;
+using sta::NetConnectedPinIterator;
+using sta::Path;
+using sta::PathExpanded;
+using sta::Pin;
+using sta::Slack;
+using sta::Slew;
+using sta::Vertex;
+using sta::VertexOutEdgeIterator;
+
+bool SizeDownMove::doMove(const Path* drvr_path,
+                          int drvr_index,
+                          Slack drvr_slack,
+                          PathExpanded* expanded,
+                          float setup_slack_margin)
```

Comment:
Make args const when possible.   (Note only do so in the definition not the declaration for by-value types).  Same in sizeup

Reply:
This interface will be changed in my next PR. The current one was copied from the moves prior to my refactor. I'd like to defer this until my next PR.